src/dynamic_bridge.cpp

```diff
@@ -196,6 +206,147 @@ void update_bridge(
     bridges_2to1.erase(topic_name);
     printf("removed 2to1 bridge for topic '%s'\n", topic_name.c_str());
   }
+
+  // create bridges for ros1 services
+  for (auto& service : ros1_services) {
+    auto& name = service.first;
+    auto& details = service.second;
+    if (
+      service_bridges_2_to_1.find(name) == service_bridges_2_to_1.end() &&
+      service_bridges_1_to_2.find(name) == service_bridges_1_to_2.end())
+    {
+      auto factory = ros1_bridge::get_service_factory("ros1", details.at("package"), details.at("name"));
+      if (factory) {
+        try {
+          service_bridges_2_to_1[name] = factory->service_bridge_2_to_1(ros1_node, ros2_node, name);
+          // printf("Created 2 to 1 bridge for service %s\n", name.data());
```

Comment:
These messages should be printed (same as for pub / sub) so that the user sees what is happening.